tests/framework/installer/environment.go

```diff
@@ -46,6 +41,17 @@ func TestIsOfficialBuild() bool {
 	return getEnvVarWithDefault("TEST_IS_OFFICIAL_BUILD", "") != "false"
 }
 
+func StorageClassName() string {
+	return getEnvVarWithDefault("TEST_STORAGE_CLASS", "")
+}
+
+func UsePVC() bool {
+	if StorageClassName() == "gp2" {
```

Comment:
The storage class could be something else. As long as it is set to something, we are using PVCs, right?
```suggestion
	if StorageClassName() != "" {
```

Reply:
Done.